provider/core/src/datagen/mod.rs

```diff
@@ -83,6 +79,12 @@ pub trait DataExporter: Sync {
     fn close(&mut self) -> Result<(), DataError> {
         Ok(())
     }
+
+    /// Returns whether the provider supports built-in fallback. If `true`, the provider must
+    /// implement [`Self::flush_with_built_in_fallback()`] for [`BuiltInFallbackMode::Standard`].
```

Comment:
It has to support `flush_with_built_in_fallback` for *all* fallback modes, up to non-exhaustiveness variant skew.